.circleci/config.yml

```diff
@@ -236,6 +286,23 @@ workflows:
           filters:
             branches:
               only: master
+      - build-goerli-image:
+          requires:
+            - build-with-lockfile
+            - integration-testing
+            - e2e-testing
+          filters:
+            branches:
+              ignore: master
+      - build-deploy-mainnet-image:
```

Comment:
Do we still need to deploy the mainnet image ?

Given that we now have !staging and !production in addition to !qa

Reply:
We still need to build it and push it (which is all this does now - it just didn't get renamed when I removed the deploy).